### PR TITLE
Update Login Tests

### PR DESCRIPTION
**Pull request by Sgiza**
Update Login Tests + PR

### CODE REVIEW

File: src/test/java/LoginTests.java [hunk 1]
```diff
@@ -19,13 +19,13 @@ public static void loginSuccessTest()
 
         driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
         driver.get("https://deens.com/login");
-        driver.findElement(By.cssSelector("#email")).click();
+        driver.findElement(By.cssSelector("#email"));
```
Comment (vladimirKorzhukov) [inline src/test/java/LoginTests.java:22]: You can remove this line because it only finds element

File: src/test/java/LoginTests.java [hunk 2]
```diff
@@ -19,13 +19,13 @@ public static void loginSuccessTest()
...
         driver.findElement(By.cssSelector("#email")).sendKeys("macarena1@gmail.com");
-        driver.findElement(By.cssSelector("#password")).click();
+        driver.findElement(By.cssSelector("#password"));
```
Comment (vladimirKorzhukov) [inline src/test/java/LoginTests.java:24]: You can remove this line because it only finds element

File: src/test/java/LoginTests.java (hunk 1 above)
Comment (Sgiza) [reply src/test/java/LoginTests.java:22]: Got it. will keep in mind!

File: src/test/java/LoginTests.java (hunk 2 above)
Comment (Sgiza) [reply src/test/java/LoginTests.java:24]: Got it. will keep in mind!

**Pull request merged**